spring-cloud-huawei-router/src/main/java/com/huaweicloud/router/client/loadbalancer/webmvc/WebMvcServiceInstanceFilter.java

```diff
@@ -51,6 +55,6 @@ public List<ServiceInstance> filter(ServiceInstanceListSupplier supplier, List<S
 
   @Override
   public int getOrder() {
-    return CANARY_ORDER;
+    return env.getProperty("spring.cloud.loadbalance.filter.router.order", int.class, -1);
```

Comment:
Use order 100, 200, or -100, -200 is better.

Reply:
fixed